fuzz/src/chanmon_consistency.rs

```diff
@@ -1116,12 +1116,16 @@ pub fn do_test<Out: test_logger::Output>(data: &[u8], out: Out) {
 				}
 
 				// Finally, make sure that at least one end of each channel can make a substantial payment.
-				assert!(
-					send_payment(&nodes[0], &nodes[1], chan_a, 10_000_000, &mut payment_id) ||
-					send_payment(&nodes[1], &nodes[0], chan_a, 10_000_000, &mut payment_id));
-				assert!(
-					send_payment(&nodes[1], &nodes[2], chan_b, 10_000_000, &mut payment_id) ||
-					send_payment(&nodes[2], &nodes[1], chan_b, 10_000_000, &mut payment_id));
+				// Unless we expected to hit a limitation of LN state machine (see comment in `send_update_fee_and_commit`)
+				if out.locked_contains(&"Outbound update_fee HTLC buffer overflow - counterparty should force-close this channel") {
+				} else {
+					assert!(
```

Comment:
This isn't the most common assertion we'll hit if we force-close the channel - we're more likely to hit some of the "bad messages, we shouldn't get `error`" or similar.

Reply:
So I interpreted "bad messages, we shouldn't get `error` as `panic"Unhandled event"`". I think that the most common assertions we're hitting for chanmon_consistency.